modin/tests/pandas/dataframe/test_default.py

```diff
@@ -586,6 +586,21 @@ def melt(df, *args, **kwargs):
     )
 
 
+# Functional test for BUG:7206
+def test_melt_duplicate_col_names():
+    if StorageFormat.get() == "Hdk":
+        pass
```

Comment:
Why?

Reply:
In the normal melt tests above we perform a dropna for the Hdk engine. I can replicate that behavior here but I wasn't sure. That's partly why it is still in draft.

I can do the same thing as the tests above if you want.

Reply:
Let's leave it as it is.